modules/rte_mis_state/rte_mis_state.module

```diff
@@ -491,7 +491,9 @@ function _rte_mis_state_invalidate_cache(EntityInterface $entity) {
 function rte_mis_state_local_tasks_alter(&$local_tasks) {
   // Check if user registration local tasks exists.
   if (array_key_exists('user.register', $local_tasks)) {
-    $local_tasks['user.register']['title'] = t('Create new School Account');
+    unset($local_tasks['user.register']);
+    unset($local_tasks['user.pass']);
+    unset($local_tasks['user.login']);
```

Comment:
Why we are doing this? Is it asked to remove, if Yes then only from this page or from all the pages?

Reply:
We are just asked to remove the location actions from the login page.

Reply:
I think the expectation here is that the local tasks should not be visible on `Login`, `Register` & `Forget Password` page.

Reply:
Please remove this block itself from all 3 pages ( Add block visibility condition )

Reply:
On password reset page and register page we don't show the Local actions.

Reply:
Done.